client/src/components/PostFeed/PostFeed.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { getPostsForHomePage, getPostsForProfile } from '../../actions/post'
+import Post from '../Post/Post'
+import Preloader from '../Preloader/Preloader'
+
+class PostFeed extends Component {
```

Comment:
Maybe we should continue to use the function components?

Reply:
<img width="476" alt="image" src="https://user-images.githubusercontent.com/48453944/71768847-fb50ee00-2f22-11ea-809e-a72cd3b66249.png">
I am using component life cycle stages to define it's origin and get posts

Reply:
Done. Replaced ComponentDidMount with UseEffect

Reply:
Thank you!